cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.v2v, pytest.mark.manual]
+
+
+@pytest.mark.tier(1)
+def test_osp_vmware_67_test_vm_name_with_punycode_characters():
+    """
+    title: OSP: vmware 67- Test VM name with Punycode characters
+    Polarion:
+        assignee: sshveta
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+        title: Test customize request security group
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan with infra map and VM with punycode chars
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_osp_kill_the_v2v_process_in_the_middle_restart_evmserverd_should_resume_migration_pos():
+    """
+    title: OSP: kill the v2v process in the middle(restart evmserverd) - should
+    resume migration post restart
+
+    Polarion:
+        assignee: ytale
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan with infra map
+            3. Start migration
+            4. restart evmserverd on appliance
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_migration_plan_with_name_which_has_all_special_character():
```

Comment:
You can remove this too b'cause [BZ1685948](https://bugzilla.redhat.com/show_bug.cgi?id=1685948)